.github/workflows/test.yml

```diff
@@ -25,14 +14,12 @@ jobs:
     runs-on: ubuntu-latest
     container:
       image: ${{ vars.CI_IMAGE }}
-    env:
-      GITHUB_PAT: ${{ secrets.GITHUB_TOKEN }}
 
     steps:
       - name: Checkout project ⬇️
         uses: actions/checkout@v4
 
-      - name: Checkout Repo Templates      
+      - name: Checkout Repo Templates ⬇️  
```

Comment:
- suggest renaming this step to Checkout templates repo
- add 
 sparse-checkout: |
            inst/validation
            scripts 
- create a variable for the dv.templates repo name at the top of the workflow

Reply:
Changed in: [bcb6e10](https://github.com/Boehringer-Ingelheim/dv.templates/pull/17/commits/bcb6e10c702e9f49808a639f11c24e9f18ed9386)